src/GeekLearning.Storage.Azure/AzureStore.cs

```diff
@@ -103,7 +68,7 @@ public AzureStore(string storeName, string connectionString, string containerNam
             blockBlob.Properties.ContentType = contentType;
             blockBlob.Properties.CacheControl = "max-age=300, must-revalidate";
```

Comment:
Maybe there is the chance to address #27 using freshly created FileProperties :)